requirements.txt

```diff
@@ -18,7 +23,7 @@ python-novaclient==2.20.0
 python-swiftclient==2.3.1
 pyyaml
 redis
-requests==2.2.1
+nose
```

Comment:
I also thought as you, but you can't.
You can check it yourself: try to `pip install CloudFerry==0.1` on clean environment. 0.1 version have not `nose` as requirements, but fabric need it for run.